content/en/docs/setup/prep/_index.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This section is right in the middle of, but seems disconnected from, the other setup section. It doesn't have any cross-references with platform setup and install material, for example, so it's unclear when it should be read and what other setup instructions to follow as a result.

Reply:
Right now, docs/setup/_index.md lists "Review pod requirements" as the first thing to do. I think we should replace that text with "Prepare your deployment" instead, pointing to the new /docs/setup/prep page.

And yes, I think we need to put keywords on the pages in /prep which will give us some See Also links to other setup content.

Related to this, I don't care much about the content we have as "setup/_index.md" vs "setup/getting-started" vs "setup/install/kubernetes". I think we could consolidate these somewhat, which would also help the flow. But let's talk about that in a separate PR.

Reply:
I think we should more-or-less get rid of setup/_index - just make it an index page - and maybe merge setup/getting-started with setup/install/kubernetes. I'm still not sure about where and how this new prep section fits into the flow, but maybe you're right, it can replace the review pod requirements step for (non getting-started) install. I think this is something we should talk about in the the next doc WG call. There are at least 2 open issues related to this: #5299 #5300